tests/foreman/api/test_repository.py

```diff
@@ -29,6 +29,8 @@
     upload_manifest,
 )
 from robottelo.constants import (
+    CUSTOM_MODULE_STREAM_REPO_2,
+    CUSTOM_MODULE_STREAM_REPO_1,
```

Comment:
Not blocking, but please sort these imports.

Reply:
good catch! :)